examples/platforms/nrf52811/README.md

```diff
@@ -182,40 +226,41 @@ For a list of all available commands, visit [OpenThread CLI Reference README.md]
 
 [CLI]: ./../../../src/cli/README.md
 
-## Logging module
+## Working with the logging module
 
 By default, the OpenThread's logging module provides functions to output logging
 information over SEGGER's Real Time Transfer (RTT).
 
-RTT output can be viewed in the J-Link RTT Viewer, which is available from SEGGER.
-The viewer is also included in the nRF Tools. To read or write messages over RTT,
-connect an nRF5 development board via USB and run the J-Link RTT Viewer.
+The RTT output can be viewed in the J-Link RTT Viewer, which is available from SEGGER.
 
-Select the correct target device (nRF52) and the target interface "SWD".
+The viewer is also included in the nRF Tools. 
+To read or write messages over RTT:
+
+1. Connect an nRF5 development board via with a USB cable. 
+2. Run the J-Link RTT Viewer.
+3. In the Specify Target Device dropdown menu, select the nRF52 device.
+4. In the Target Interface & Speed dropdown menu, select the SWD interface.
 
 The intended log level can be set using `OPENTHREAD_CONFIG_LOG_LEVEL` define.
 
 ## Disabling the Mass Storage Device
 
-Due to a known issue in Segger’s J-Link firmware, depending on your version, you might experience data corruption or drops if you use the serial port. You can avoid this issue by disabling the Mass Storage Device:
-
- - On Linux or macOS (OS X), open JLinkExe from the terminal.
- - On Microsoft Windows, open the J-Link Commander application.
+Depending on your software version, a known issue in SEGGER's J-Link firmware can cause data corruption or data drops if you use the serial port. You can avoid this issue by disabling the Mass Storage Device:
 
-Run the following command: `MSDDisable`
+ - On Linux or macOS (OS X), open JLinkExe from the terminal and run the command `MSDDisable`.
+ - On Microsoft Windows, open the J-Link Commander application and run the command `MSDDisable`.
 
 ## Diagnostic module
 
-nRF52811 port extends [OpenThread Diagnostics Module][DIAG].
+nRF52811 supports, with some additional features [OpenThread Diagnostics Module][DIAG].
```

Comment:
Edit the order:

nRF52811 supports [OpenThread Diagnostics Module][DIAG], with some additional features.

Reply:
fixed